src/Components/dashboard/Dashboard.css

```diff
@@ -10,14 +10,24 @@
 }
 
 .containt {
-  overflow-y: scroll;
```

Comment:
Are you sure we do not need this scrolling behavior?

Reply:
This on scroll on the UI is displaying a duplicate so It is looking great with on scroll from the main style.css.